### PR TITLE
Mark auth provider icons as decorative

### PR DESCRIPTION
**Pull request by KevSlashNull**
## Description

This marks the authentication provider icons (GitHub, GitLab, Bitbucket) on the login and new project pages as [decorative](https://www.w3.org/WAI/tutorials/images/decorative/) by setting `alt=""` on the `img` element.

Setting an alternative text is good for icons but the authentication provider icons are currently directly paired with the UI text `Continue with {provider}` where `{provider}` is one of the providers, so screen readers would announce `GitLab Continue with GitLab`, which is currently redundant.

## Relevant section

image

No visual changes intended.

## Related Issue(s)

#3841 for the broader effort to reduce warnings

## How to test

1. Go to the login page
2. Open your browser’s DevTools
3. Use the element picker
4. Click on one of the authentication provider icons
5. See that the `alt` attribute is set to an empty string

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by geropl — APPROVED** [review]
LGTM!

**Comment by mads-hartmann** [comment]
/hold
CI is not passing as the job never ran (see comment above [here](https://github.com/gitpod-io/gitpod/pull/15590#issuecomment-1372752438)). Holding to unblock the merge queue

File: components/dashboard/src/provider-utils.tsx
Comment (Siddhant-K-code) [inline components/dashboard/src/provider-utils.tsx:15]: alt can be filled out too 😅 

`alt="GitHub"`

Comment (gtsiolis) [reply components/dashboard/src/provider-utils.tsx:15]: Absolutely valid point, @Siddhant-K-code! Could you open a follow-up issue or PR for this?

Comment (Siddhant-K-code) [reply components/dashboard/src/provider-utils.tsx:15]: I've created this PR: https://github.com/gitpod-io/gitpod/pull/15617

**Pull request merged**
